endpoints/_includes/functions.php

```diff
@@ -827,6 +855,8 @@ function getResultsString($filtered_list)
 
 function getServiceBodyCoverage($latitude, $longitude)
 {
+    getBMLTServer($latitude, $longitude, SearchType::VOLUNTEERS);
```

Comment:
Should we be doing something with the output from this function?

Reply:
At that particular location no. That is just to run a check on conditions to see if tomato_helpline_routing is needed to be flipped on or not. So the return output from the function is not needed there.

Reply:
A function should do one thing, and do it well.  The way I'm reading this code, the function does two things, returns a value and also changes a state from inside the function.  My suggestion is to decouple these behaviors.

Reply:
I can break out the rootserver verification into a separate function easily enough.